client/hosting/overview/components/plan-card.tsx

```diff
@@ -195,6 +195,30 @@ const PricingSection: FC = () => {
 	);
 };
 
+function PlanStorageFooter( { children }: PropsWithChildren< unknown > ) {
```

Comment:
Hm, looks like we don't need it
```suggestion
function PlanStorageFooter( { children }: PropsWithChildren ) {
```